VideoLocker/src/main/java/org/edx/mobile/http/OkHttpUtil.java

```diff
@@ -21,13 +24,27 @@
 import java.util.Date;
 import java.util.List;
 
-
-/**
- * Created by hanning on 6/15/15.
- */
 public class OkHttpUtil {
     public static final MediaType JSON
         = MediaType.parse("application/json; charset=utf-8");
+
+
+    private static final int cacheSize = 10 * 1024 * 1024; // 10 MiB
+
+    public static OkHttpClient getOAuthBasedClient(@NonNull Context context) {
+        OkHttpClient client = new OkHttpClient();
+        File cacheDirectory = new File(context.getFilesDir(), "http-cache");
+        if (!cacheDirectory.exists()) {
+            cacheDirectory.mkdirs();
+        }
+        Cache cache = new com.squareup.okhttp.Cache(cacheDirectory, cacheSize);
```

Comment:
The `com.squareup.okhttp` namespace can be removed, as we have imported `Cache` here.